Defs/FactionDefs/Factions_Zombiefied.xml

```diff
@@ -13,20 +13,17 @@
 				<li>(0, 0)</li>
 			</points>
 		</raidCommonalityFromPointsCurve>
+		<raidLootValueFromPointsCurve>
+			<points>
+				<li>(0, 0)</li>
+			</points>
+		</raidLootValueFromPointsCurve>
     <humanlikeFaction>false</humanlikeFaction>
     <hidden>true</hidden>
     <autoFlee>false</autoFlee>
     <canUseAvoidGrid>false</canUseAvoidGrid>
     <techLevel>Animal</techLevel>
-    <startingGoodwill>
-      <min>-100</min>
-      <max>-100</max>
-    </startingGoodwill>
-    <mustStartOneEnemy>true</mustStartOneEnemy>
-    <naturalColonyGoodwill>
-      <min>-100</min>
-      <max>-100</max>
-    </naturalColonyGoodwill>
+		<permanentEnemy>true</permanentEnemy>
```

Comment:
Is there a reason for this line <permanentEnemy>? Might break interactions with some factions

Reply:
yeah, `naturalColonyGoodwill` was removed, according to some other modders that I was talking to there were a few other options, including one that's "enemy to all factions except ..." which takes a list. Not sure how factions are really used in this mod.